DESCRIPTION

```diff
@@ -72,3 +72,5 @@ Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
 Config/testthat/edition: 3
+Remotes: 
+    r-lib/generics
```

Comment:
We are going to wait until generics hits cran to merge this, and we will update to use CRAN generics once that happens